README.md

```diff
@@ -6,13 +6,16 @@
 
 Build it:
 ```
-$ docker build -t trackerbot . 
+docker build -t trackerbot .
 ```
+
 Run it interactive:
 ```
-$ docker run -itv $(pwd):/app --name tb trackerbot
+docker run -itv $(pwd):/app --name tb trackerbot
 ```
+
 Run tests inside the container:
 ```
-$ npm run test
+npm install
```

Comment:
`npm install` is needed because in the Dockerfile only the `--production` dependencies are installed.